### ISSUE TITLE
bug(mat-button): Focus is barely perceptible on some buttons styles (accessibility issue)

### ISSUE THREAD

**Issue by nloding** [angular/components#19195]
#### Reproduction

StackBlitz demo: https://stackblitz.com/edit/components-issue-gvtqjt

Steps to reproduce:
1. Tab between each button
2. Notice the styles applied
 
 
#### Expected Behavior

A visible and clear indication of which element has been focused.

#### Actual Behavior

The `mat-raised-button` with no color, and the regular `mat-button`, have very clear style changes indicating they are focused. The `primary` raised button is _barely_ visible. Other button styles have visible changes but may not meet proposed contrast requirements.

#### Environment

- Angular: ^9.0.2
- CDK/Material: ^9.1.0 / 
- Browser(s): Firefox, Chrome, Edge
- Operating System (e.g. Windows, macOS, Ubuntu):  Windows 10, MacOS Catalina

#### Notes

It may also be worth looking ahead to the proposed WCAG 2.2 guidelines, which have much clearer requirements regarding contrast. https://www.w3.org/WAI/WCAG21/Understanding/focus-visible-enhanced

**Comment by nloding** [comment]
In reviewing the Material Design spec, it looks like a good fix may be to use a white overlay on darker elements (see the "Overlay Opacity Values" example #2). This may not work properly with high contrast modes, however.

**Comment by jelbourn** [comment]
This should be addressed when we switch to a button based on top of MDC Web

**Comment by amysorto** [comment]
The new MDC based components that are going to be available in v15 have improvements to the button focus styles. For example, the primary raised button now has a slight white overlay and the shadow increases when you tab to it. Closing this bug.
